CHANGELOG.md

```diff
@@ -4,6 +4,27 @@ This component provides official bindings to the Urban Airship SDK, as well as s
 
 ### Release Notes
 
+Version 5.1.0 - May 18, 2018
+=============================
+- Updated iOS SDK to 9.1.0
+- Updated Android SDK to 9.2.0.
+
+Note: Android SDK version 9.1.0 introduced support for FCM apis, modular packages, and in-app message design updates. More Android-specific details are available in the [changelog](https://github.com/urbanairship/android-library/blob/9.1.0/CHANGELOG.md). For FCM migration, please follow the [FCM Migration Guide](https://github.com/urbanairship/xamarin-component/blob/master/documentation/migration/migration-guide-fcm.md).
+
+Changes
+-------
+- Moved Android push providers into their own packages. `urbanairship` has been split into:
+
+   - `urbanairship.core`
```

Comment:
should this be android.core?

Reply:
`urbanairship.core` contains iOS and Android.Core bindings. It's basically what used to be `urbanairship` with the push providers extracted.